tests/integration/test_wazuh_db/data/agent/syscollector_deltas_messages.yaml

```diff
@@ -0,0 +1,297 @@
+---
+-
+  name: 'miscellaneous'
```

Comment:
where is this file referenced?

Reply:
Hi @CamiRomero ,
The file `syscollector_deltas_messages.yaml` contains different input/expected output messages grouped by the semantic s they try to validate. In case of deltas, the were divided by the type of inventory data: ports, hardware, osinfo, processes, and so. `miscellaneous` tries to group some test that does not match with any of those types.
Please let me know if i was clear enough